packages/api/tests/endpoints/collections/create-collection.js

```diff
@@ -558,7 +555,7 @@ test.serial('POST does not write to Elasticsearch/SNS if writing to PostgreSQL f
     WaitTimeSeconds: 10,
   }).promise();
 
-  t.is(Messages, undefined);
```

Comment:
Was this a result of a fix in localstack getting their typing right?   Yay localstack 🎉

Reply:
yes